src/relations/db.py

```diff
@@ -250,11 +250,8 @@ def _on_relation_broken(self, event: RelationBrokenEvent) -> None:
             self.charm._has_blocked_status
             and self.charm.unit.status.message == EXTENSIONS_BLOCKING_MESSAGE
         ):
-            if "extensions" in event.relation.data.get(
-                event.app, {}
-            ) or "extensions" in event.relation.data.get(event.unit, {}):
```

Comment:
ops 1.5.4 test harness complains if we access the relationship databag during `_on_relation_broken`. Since we enter this block only when we block due to enabled extensions, it should be OK to remove.